src/panels/beaconSummaryPanel/BeaconSummaryPanel.tsx

```diff
@@ -0,0 +1,141 @@
+import { Card, CardContent, CardHeader } from "@material-ui/core";
+import { FunctionComponent, useEffect, useState } from "react";
+import { EditPanelButton } from "../../components/dataPanel/EditPanelButton";
+import { ErrorState } from "../../components/dataPanel/PanelErrorState";
+import { LoadingState } from "../../components/dataPanel/PanelLoadingState";
+import { PanelViewingState } from "../../components/dataPanel/PanelViewingState";
+import { DataPanelStates } from "../../components/dataPanel/States";
+import { IBeacon } from "../../entities/IBeacon";
+import { IBeaconsGateway } from "../../gateways/IBeaconsGateway";
+import {
+  formatEmergencyContacts,
+  formatOwners,
+  formatUses,
+  Placeholders,
+} from "../../useCases/mcaWritingStyleFormatter";
+import { EditingState } from "./EditingState";
+
+interface IBeaconSummaryProps {
+  beaconsGateway: IBeaconsGateway;
+  beaconId: string;
+}
+
+export const BeaconSummaryPanel: FunctionComponent<IBeaconSummaryProps> = ({
+  beaconsGateway,
+  beaconId,
+}): JSX.Element => {
+  const [beacon, setBeacon] = useState<IBeacon>({} as IBeacon);
+  const [userState, setUserState] = useState<DataPanelStates>(
+    DataPanelStates.Viewing
+  );
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(true);
```

Comment:
Separated out `userState` (viewing, editing) from component state (error, loading) because they are subtly separate concerns.  This allows the `useEffect` hook to control loading/error separately of what the user is doing.

(I'm not sure `userState` is the best name and would appreciate your thoughts.)

If we get much more complicated than this there are libraries made for managing React state machines we might consider (e.g. https://xstate.js.org/docs/, 840k weekly downloads).